.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
not sure we want this file

Reply:
it's a pain not having this if you have formatOnSave enabled by default since blockly seems to use some formatting settings vscode doesn't understand.
Should be safe to check this in, no?